packages/app/src/runner/StudioControls.vue

```diff
@@ -0,0 +1,118 @@
+<template>
+  <div v-if="!studioStore.url && studioStore.isActive">
```

Comment:
We are missing tests for this component.

Reply:
Yep, thanks for calling this out -- DM'd you, gonna merge this to the feature branch and add a bunch of tests here:  https://github.com/cypress-io/cypress/issues/23461